src/settings.js

```diff
@@ -1646,6 +1656,15 @@ var ASAN_SHADOW_SIZE = 33554432;
 // entries across function boundaries.
 var USE_OFFSET_CONVERTER = 0;
 
+// Maximum seen global alignment; received from the backend.
+var MAX_GLOBAL_ALIGN = -1;
+
+// List of functions implemented in compiled code; received from the backend.
+var IMPLEMENTED_FUNCTIONS = [];
+
+// Name of the file containing the Fetch *.fetch.js, if relevant
+var FETCH_WORKER_FILE = '';
```

Comment:
Here is no internal area anymore!  I moved these to `settings_internal.js`.

Reply:
Hmm, this file (before this PR) does still have a quite large internal section?

https://github.com/emscripten-core/emscripten/blob/0fae35adbd4e013cc5d46a7d09283087334c634a/src/settings.js#L1490

Is that all in the wrong place?

Reply:
I converted a lot of them over when I did the initial transition in #9649.. Perhaps the remaining ones were actually still used in some places or I wasn't 100 sure.   But yes, any truly internal settings here are in the wrong place.

Reply:
I see. Ok, for this PR, I just moved those 3 that already moved to the right place.